b2b_ecommerce/constants.py

```diff
@@ -0,0 +1,2 @@
+"""Constants for b2b_ecommerce"""
+REFERENCE_NUMBER_PREFIX = "XPRO-ENROLLMENT-"
```

Comment:
Still think it wouldn't hurt to change this to be a little less arbitrary. `MITXPRO-B2B-` maybe